taskfile/node_git.go

```diff
@@ -40,13 +40,10 @@ func NewGitNode(
 		return nil, err
 	}
 
-	basePath, path := func() (string, string) {
-		x := strings.Split(u.Path, "//")
-		return x[0], x[1]
-	}()
+	basePath, path := splitPath(u)
 	ref := u.Query().Get("ref")
 
-	rawUrl := u.String()
+	rawUrl := u.Redacted()
```

Comment:
This redacted version should be used as the node.URL and for the `errors.TaskfileNotSecureError.URI`,

Reply:
Thanks, I missed the `errors.TaskfileNotSecureError.URI`